### DISCUSSION TITLE
container updates (master or child once) does not remove old images after completion

### DISCUSSION THREAD

**Discussion by fate8383** [nextcloud/all-in-one#1727] [Ideas] [answered]
as the title says, old images of the containers should be deleted, average user will not delete that by default just using UI and hitting update leaves on the back all the old images. Not sure if this is needed from a restore/backup perspective but I don't think so. Any comments? I manually prune after each update. Thanks

**Comment by szaimen** [comment, marked as answer]
Hi, we have a cronjob for this that runs once a day.
